src/services/tags/create-tag.test.js

```diff
@@ -0,0 +1,51 @@
+const {expect} = require('chai')
+const { faker } = require('@faker-js/faker')
+const db = require("../../db")
+const createTag = require("./create-tag")
+
+
+describe('Create tag', () => {
+	
+	let tag
+	before(async ()=>{
+		  
+		 tag = await createTag({
+			title : faker.random.word(5),
+			content : faker.random.words()
+		})
+		 
+	})
+
+	it('should return an error when tag already exists', async () => 
+		
+		expect(createTag(tag))
```

Comment:
not sure I get how this test case works yet. not seeing where the same tag is being inserted twice

Reply:
the tag is in the before

Reply:
seen it, but similar to what I said in the endpoint tests, we're testing the `createTag` itself if it works

we can't call the same function we're testing twice because we don't even know if its inserting the tag in the first place. That's why we use the utils to set up the tests

Reply:
yeah, i have taken note of that

Reply:
done